app/src/main/java/com/x8bit/bitwarden/ui/autofill/fido2/manager/Fido2CompletionManagerImpl.kt

```diff
@@ -84,4 +95,56 @@ class Fido2CompletionManagerImpl(
             it.finish()
         }
     }
+
+    override fun completeFido2GetCredentialRequest(result: Fido2GetCredentialResult) {
+        val entries = mutableListOf<CredentialEntry>()
+        val resultIntent = Intent()
+        val responseBuilder = BeginGetCredentialResponse.Builder()
+        when (result) {
+            is Fido2GetCredentialResult.Success -> {
+                result
+                    .credentials
+                    .onEach { credential ->
+                        val credentialIntent = Intent(GET_PASSKEY_INTENT)
+                            .setPackage(activity.packageName)
+                            .putExtra(EXTRA_KEY_CIPHER_ID, credential.cipherId)
+                            .putExtra(EXTRA_KEY_CREDENTIAL_ID, credential.credentialId.toString())
+                        val pendingIntent = PendingIntent.getActivity(
+                            activity,
+                            Random.nextInt(),
```

Comment:
A lot of this looks super similar to the method `createFido2GetCredentialPendingIntent` in the `IntentManager`, but I'm guessing there's a reason we can't reuse / consolidate it?